README.md

```diff
@@ -1,18 +1,49 @@
 # vets-api-clients
 
-This repository contains resources for accessing vets-api as a third-party
-client. It includes API specifications, sample clients, and general
-access information.
+Welcome to the public-facing repository for **Department of Veterans Affairs (VA) Lighthouse APIs**.  For the most up-to-date API documentation, please visit [developer.va.gov/explore](https://developer.va.gov/explore).
 
-**Third-party access to vets-api is currently in limited beta status, and
-is only available in development environments.**
+If you would like to improve or suggest **edits to API documentation** for the VA APIs, please visit [this repo](https://github.com/department-of-veterans-affairs/developer-portal/tree/master/src/content/apiDocs).
```

Comment:
I would point them here (https://github.com/department-of-veterans-affairs/vets-api) for the APIs repo and here (https://github.com/department-of-veterans-affairs/developer-portal) for the Developer Portal repo. What could be considered "API documentation" can be found in both places.